people-and-planet-ai/README.md

```diff
@@ -45,6 +45,8 @@ location data.
 
 [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/GoogleCloudPlatform/python-docs-samples/blob/main/people-and-planet-ai/geospatial-classification/README.ipynb)
 
+> [Watch the video in YouTube<br> ![thumbnail](http://img.youtube.com/vi/8amFK7T_n30/0.jpg)](https://youtu.be/8amFK7T_n30)
```

Comment:
Thanks, do you mind adding this into the `people-and-planet-ai/geospatial-classification/README.md` file as well?

Reply:
Done!